src/commands/secrets/upload.ts

```diff
@@ -38,13 +45,22 @@ export default class SecretsUpload extends BaseCommand {
 
   async run(): Promise<void> {
     const { flags, args } = await this.parse(SecretsUpload);
-    const account = await AccountUtils.getAccount(this.app, flags.account);
+    if (!flags.account) {
+      this.log(chalk.red('Account is not found. Please see examples below:'));
+      this.log(`  ${SecretsUpload.examples.join('\n  ')}`);
+      return;
```

Comment:
We want to exit with a non 0 status code here since the operation itself failed. `this.exit(1)` should work or just throwing an `ArchitectError` that is not tracked.